### PR TITLE
Aviation Lights changed ownership.

### PR DESCRIPTION
**Pull request by Lisias**
- <https://spacedock.info/mod/3208>
- <https://forum.kerbalspaceprogram.com/index.php?/topic/211781-*/>
- <https://github.com/net-lisias-ksp/AviationLights/>

See https://forum.kerbalspaceprogram.com/index.php?/topic/173305-112x-aviation-lights-v413-29-january-2023-available-for-adoption/&do=findComment&comment=4225479

### CODE REVIEW

**Comment by Lisias** [comment]
@HebaruSan , there's something wrong with this pull request?

**Comment by HebaruSan** [comment]
> @HebaruSan , there's something wrong with this pull request?

Yes, the `$kref` field is invalid. It references both github and spacedock, which causes the attempt to download to fail:

```
  Error: 1668 [1] FATAL CKAN.NetKAN.Program (null) - The remote server returned an error: (404) Not Found.
```

File: NetKAN/AviationLights.netkan
Comment (HebaruSan) [inline NetKAN/AviationLights.netkan:6]:
```suggestion
    "$kref":        "#/ckan/spacedock/3208",
```

Comment (Lisias) [reply NetKAN/AviationLights.netkan]: UGH… I just fixed it on a commit below...

**Comment by Lisias** [comment]
@HebaruSan , there's something else in need to be fixed?

Thanks.

File: NetKAN/AviationLights.netkan
Comment (HebaruSan) [inline NetKAN/AviationLights.netkan:25]: Please leave this override in, it's nice to have for historical purposes

File: NetKAN/AviationLights.netkan
Comment (HebaruSan) [inline NetKAN/AviationLights.netkan:13]:
```suggestion
```

You don't need this, as it will be pulled from SpaceDock automatically

**Review by HebaruSan — CHANGES_REQUESTED** [review]
Added some comments, and it would be nice to YAMLize this while we're looking at it:
<https://codebeautify.org/json-to-yaml>

File: NetKAN/AviationLights.netkan
Comment (Lisias) [reply NetKAN/AviationLights.netkan]: fixed on commit https://github.com/KSP-CKAN/NetKAN/pull/9550/commits/7c84a0deca8c30f939ce14ce993b2ad1da97179f

File: NetKAN/AviationLights.netkan
Comment (Lisias) [reply NetKAN/AviationLights.netkan]: fixed on commit https://github.com/KSP-CKAN/NetKAN/pull/9550/commits/838d78dc4fe2afdb6c7fbb8243a8df28cc7ddfb8

**Comment by Lisias** [comment]
> Added some comments, and it would be nice to YAMLize this while we're looking at it:
> https://codebeautify.org/json-to-yaml

Done on commit https://github.com/KSP-CKAN/NetKAN/pull/9550/commits/e16c13759821cfb004a0ef237e42ae0d193679c4

**Comment by Lisias** [comment]
Hi, @HebaruSan ! There's something else in need to be changed in order to merge this P/R?

Thanks!

**Comment by Lisias** [comment]
Hi, @HebaruSan !

There's something else in need to be fixed before merging?

Thanks!

**Comment by HebaruSan** [comment]
Previous maintainer's official hand-off 👍:

https://forum.kerbalspaceprogram.com/index.php?/topic/173305-112x-aviation-lights-v413-29-january-2023-available-for-adoption/&do=findComment&comment=4225479

image

**Review by HebaruSan — CHANGES_REQUESTED** [review]
This warning indicates there is ModuleManager syntax in these files:

> `2839 [1] WARN CKAN.NetKAN.Validators.ModuleManagerDependsValidator (null) - ModuleManager syntax used without ModuleManager dependency: GameData/AviationLights/Parts/aviation_light.cfg, GameData/AviationLights/Parts/lights/lightbeacon_amber.cfg, GameData/AviationLights/Parts/lights/lightbeacon_red.cfg, GameData/AviationLights/Parts/lights/lightnav_blue.cfg, GameData/AviationLights/Parts/lights/lightnav_green.cfg, GameData/AviationLights/Parts/lights/lightnav_red.cfg, GameData/AviationLights/Parts/lights/lightnav_white.cfg, GameData/AviationLights/Parts/lights/lightstrobe_white.cfg, GameData/AviationLights/Patches/MM_KIS.cfg, GameData/AviationLights/Patches/MM_UnhideOldLights.cfg`

Can you add a relationship to account for this?

```yaml
depends:
  - name: ModuleManager
```

It's only a warning because sometimes there are reasons not to include it, but if that isn't the case here and a user selects _only_ this mod to install, then it would be good for these files to work as intended.

**Comment by Lisias** [comment]
Done!

Let me know if I need to change something else!

**Review by HebaruSan — APPROVED** [review]
Thanks, the warning is gone and everything else looks fine now. 👍

**Pull request merged**
